zetaclient/bitcoin_client.go

```diff
@@ -460,7 +459,10 @@ func (ob *BitcoinChainClient) PostGasPrice() error {
 		return fmt.Errorf("error getting gas price: %s", feeResult.Errors)
 	}
 	gasPrice := big.NewFloat(0)
-	gasPriceU64, _ := gasPrice.Mul(big.NewFloat(*feeResult.FeeRate), big.NewFloat(1e8)).Uint64()
+	// feerate from RPC is BTC/KB, convert it to satoshi/byte
+	// FIXME: in zetacore the gaslimit(vsize in BTC) is 100 which is too low for a typical outbound tx
+	// until we fix the gaslimit in BTC, we need to multiply the feerate by 20 to make sure the tx is confirmed
+	gasPriceU64, _ := gasPrice.Mul(big.NewFloat(*feeResult.FeeRate), big.NewFloat(20*1e5)).Uint64()
```

Comment:
I don't think we should bring this change to `develop` now that we can update gas limit

Reply:
you are right, we hold off merging to develop until we remove this; 
we can remove this adjustment after MsgUpdate gas limit of bitcoin chain